### ISSUE TITLE
Unsupported Launch options not working

### ISSUE THREAD

**Issue by chocolatechipcats** [felixkmh/StartPage-for-Playnite#154]
As of 2.8.0, the Unsupported options 'Sidebaritem activated on launch' and 'Move Sidebar Item to Top of Sidebar on Launch' both fail for me. 

I tried deactivating and reactivating them both individually.

Can anyone else confirm?

**Comment by felixkmh** [comment]
Can you find anything in the playnite.log? I added some checkpoints that are logged in the extensions.log file with this build:
[felixkmh_StartPage_Plugin_2_8_0.zip](https://github.com/felixkmh/StartPage-for-Playnite/files/8962888/felixkmh_StartPage_Plugin_2_8_0.zip)
Can you check what is logged when trying this build?

**Comment by chocolatechipcats** [comment]
``22-06 22:37:14.137|DEBUG|StartPagePlugin#LandingPageExtension:Trying to execute unsupported actions...
22-06 22:37:14.262|DEBUG|StartPagePlugin#LandingPageExtension:Could not find StartPage side bar button.``

**Comment by felixkmh** [comment]
I don't understand why this would not work after the update. I dont think I changed anything related to this functionality.
Try looking the logs with this:
[felixkmh_StartPage_Plugin_2_8_0.zip](https://github.com/felixkmh/StartPage-for-Playnite/files/8962955/felixkmh_StartPage_Plugin_2_8_0.zip)

**Comment by felixkmh** [comment]
[felixkmh_StartPage_Plugin_2_8_0.zip](https://github.com/felixkmh/StartPage-for-Playnite/files/8963005/felixkmh_StartPage_Plugin_2_8_0.zip)

**Comment by chocolatechipcats** [comment]
That seems to have fixed it. Here's the log for documentation purposes

```22-06 22:58:13.742|DEBUG|StartPagePlugin#LandingPageExtension:Found side bar panel PART_PanelSideBarItems with 8 children of types SidebarItem.
22-06 22:58:13.742|DEBUG|StartPagePlugin#LandingPageExtension:Found side bar items:
Library
Statistics
Achievements
RetroAchievements
Start Page
View games activities
FRAPS
Savegame Folder```

**Comment by felixkmh** [comment]
Not quiete yet
[felixkmh_StartPage_Plugin_2_8_0.zip](https://github.com/felixkmh/StartPage-for-Playnite/files/8963037/felixkmh_StartPage_Plugin_2_8_0.zip)

**Comment by felixkmh** [comment]
sorry, wrong file
[felixkmh_StartPage_Plugin_2_8_0.zip](https://github.com/felixkmh/StartPage-for-Playnite/files/8963048/felixkmh_StartPage_Plugin_2_8_0.zip)

**Comment by chocolatechipcats** [comment]
> sorry, wrong file [felixkmh_StartPage_Plugin_2_8_0.zip](https://github.com/felixkmh/StartPage-for-Playnite/files/8963048/felixkmh_StartPage_Plugin_2_8_0.zip)

```22-06 23:12:42.069|DEBUG|ItchioLibrary#ItchioLibrarySettingsViewModel:Updating itch settings from version 0.
22-06 23:12:44.393|INFO |CheckDlc#PluginDatabaseObject`4:LoadDatabase with 411 items - 00:02.17
22-06 23:12:44.393|INFO |GameActivity#PluginDatabaseObject`4:LoadDatabase with 418 items - 00:02.04
22-06 23:12:44.393|INFO |SuccessStory#PluginDatabaseObject`4:LoadDatabase with 410 items - 00:01.88
22-06 23:12:44.816|DEBUG|StartPagePlugin#LandingPageExtension:Trying to execute unsupported actions...
22-06 23:12:44.931|DEBUG|StartPagePlugin#LandingPageExtension:Found side bar panel PART_PanelSideBarItems with 8 children of types SidebarItem.
22-06 23:12:44.931|DEBUG|StartPagePlugin#LandingPageExtension:Found side bar items:

22-06 23:12:44.931|DEBUG|StartPagePlugin#LandingPageExtension:Could not find StartPage side bar button.```

**Comment by felixkmh** [comment]
If this works, I'll consider this fixed:
[felixkmh_StartPage_Plugin_2_8_0.zip](https://github.com/felixkmh/StartPage-for-Playnite/files/8963094/felixkmh_StartPage_Plugin_2_8_0.zip)

**Comment by chocolatechipcats** [comment]
Sadly I'm getting the same error as before :(

https://github.com/felixkmh/StartPage-for-Playnite/issues/154#issuecomment-1163837964 This version works

**Comment by felixkmh** [comment]
Yeah, because I'm waiting until Playnite is idle before doing anything. That could take some time depending on the setup, so I don't like that solution.

**Comment by chocolatechipcats** [comment]
Yeah, I noticed there was a slight delay before the icons rearranged themselves. 

Should I try removing the two software tools (FRAPS and the savegame folder) from the sidebar?

**Comment by felixkmh** [comment]
Try this and check the log again:
[felixkmh_StartPage_Plugin_2_8_0.zip](https://github.com/felixkmh/StartPage-for-Playnite/files/8963140/felixkmh_StartPage_Plugin_2_8_0.zip)

**Comment by chocolatechipcats** [comment]
It worked, with a slight delay:

```22-06 23:34:37.105|DEBUG|ItchioLibrary#ItchioLibrarySettingsViewModel:Updating itch settings from version 0.
22-06 23:34:41.174|INFO |CheckDlc#PluginDatabaseObject`4:LoadDatabase with 411 items - 00:03.88
22-06 23:34:41.174|INFO |GameActivity#PluginDatabaseObject`4:LoadDatabase with 418 items - 00:03.50
22-06 23:34:41.191|INFO |SuccessStory#PluginDatabaseObject`4:LoadDatabase with 410 items - 00:02.83
22-06 23:34:41.623|DEBUG|StartPagePlugin#LandingPageExtension:Trying to execute unsupported actions...
22-06 23:34:41.727|DEBUG|StartPagePlugin#LandingPageExtension:Could not find any side bar items. Waiting 100ms and retrying. Remaining retries: 20.
22-06 23:34:42.107|DEBUG|StartPagePlugin#LandingPageExtension:Found side bar items:
Library
Statistics
Achievements
RetroAchievements
Start Page
View games activities
FRAPS
Savegame Folder```

**Comment by felixkmh** [comment]
Sounds good. The delay should be minimal. I will release the fix with v2.8.1.
